commit 99620810e5245f319986d58a49ff13067814ef5a
Author: Michael Biebl <biebl@debian.org>
Date:   Thu Feb 13 23:01:47 2014 +0100

    Fix json includes
    
    Rely on the include paths set by json-c.pc instead of hardcoding
    the include to be <json/json.h> since this breaks with newer versions of
    json-c where the files are installed in /usr/include/json-c.
    
    While at it, remove a duplicate include.

diff --git a/plugins/mmcount/mmcount.c b/plugins/mmcount/mmcount.c
index 56a4de5..8a40e38 100644
--- a/plugins/mmcount/mmcount.c
+++ b/plugins/mmcount/mmcount.c
@@ -30,7 +30,7 @@
 #include <errno.h>
 #include <unistd.h>
 #include <stdint.h>
-#include <json/json.h>
+#include <json.h>
 #include "conf.h"
 #include "syslogd-types.h"
 #include "srUtils.h"
diff --git a/plugins/mmsequence/mmsequence.c b/plugins/mmsequence/mmsequence.c
index 20a8537..47e9d5f 100644
--- a/plugins/mmsequence/mmsequence.c
+++ b/plugins/mmsequence/mmsequence.c
@@ -35,7 +35,7 @@
 #include <stdint.h>
 #include <time.h>
 #include <limits.h>
-#include <json/json.h>
+#include <json.h>
 #include <pthread.h>
 #include "conf.h"
 #include "syslogd-types.h"
diff --git a/runtime/lookup.c b/runtime/lookup.c
index bc3b1a9..f2af17b 100644
--- a/runtime/lookup.c
+++ b/runtime/lookup.c
@@ -26,8 +26,7 @@
 #include <sys/types.h>
 #include <sys/stat.h>
 #include <fcntl.h>
-#include <json/json.h>
-#include <json/json.h>
+#include <json.h>
 #include <assert.h>
 
 #include "rsyslog.h"
